sdk/policyinsights/Azure.ResourceManager.PolicyInsights/src/Customized/Models/SlimPolicyMetadata.cs

```diff
@@ -49,7 +50,12 @@ internal SlimPolicyMetadata(ResourceIdentifier id, string name, ResourceType res
         /// <summary> The owner of the policy metadata. </summary>
         public string Owner { get; }
         /// <summary> Url for getting additional content about the resource metadata. </summary>
-        public Uri AdditionalContentUri { get; }
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Uri AdditionalContentUri => Uri.IsWellFormedUriString(AdditionalContentUriString, UriKind.Absolute) ? new Uri(AdditionalContentUriString) : null;
```

Comment:
This is readonly property, so just parse it in the constructor, not here.

Reply:
![image](https://user-images.githubusercontent.com/61542202/230327526-01f3a323-0812-49fb-b6d9-b123a27b13ee.png)
```
        [Obsolete("This property has been replaced by AdditionalContentUriString", true)]
        public Uri AdditionalContentUri { get; }
```
if we add the [Obsolete] attribute, it cannot be used otherwhere